spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/impl/DefaultSchedulerService.java

```diff
@@ -182,13 +211,16 @@ public void schedule(String scheduleName, String taskDefinitionName, Map<String,
 			taskDefinition = new TaskDefinition(taskDefinition.getName(),
 					TaskServiceUtils.createComposedTaskDefinition(
 							taskNode.toExecutableDSL()));
-			taskDeploymentProperties = TaskServiceUtils.establishComposedTaskProperties(taskDeploymentProperties, taskNode);
+			taskDeploymentProperties = TaskServiceUtils.establishComposedTaskProperties(taskDeploymentProperties,
```

Comment:
Looks like your formatter has a max limit per line.   Please remove that restriction.   Thanks.